sklearn/datasets/_kddcup99.py

```diff
@@ -189,7 +224,7 @@ def _fetch_brute_kddcup99(data_home=None,
         all scikit-learn data is stored in '~/scikit_learn_data' subfolders.
 
     download_if_missing : bool, default=True
-        If False, raise a IOError if the data is not locally available
+        If False, raise a OSError if the data is not locally available
```

Comment:
What is the reason for this change?

If we are doing this, lets have another PR do this with all the `fetch_*` functions.

Reply:
Hey @thomasjpfan sorry for the late reply; i remember this is due to a test case not passing. It might not have been the right approach here. I will take a closer look this week.